services/s3/pom.xml

```diff
@@ -224,5 +224,17 @@
             <artifactId>jimfs</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
We should probably avoid circular dependency in the SDK. It'll likely cause more dependency issues in the future. (Side note, I didn't know you could use exclusions trick to work around the issue)

Reply:
removed integ test, one-off testing and unit test should be sufficient